controllers/AuthController.js

```diff
@@ -0,0 +1,58 @@
+const bcrypt = require('bcrypt');
+const userModelManager = require('../lib/modelManagers/usermodel');
+const jwtUtil = require('../lib/utils/jwtUtil');
+
+/** Class to add new user and login registered user. */
+class AuthController {
+  /**
+   * Create a User.
+   * @param {object} req - The request sent by user.
+   * @param {object} res - The request sent by user.
+   * @param {function} next - The next function to send email to user.
+   * @returns {object} The status of response and body of the response.
+   */
+  static async signUp(req, res, next) {
+    try {
+      // Hash password before sending
+      const hashedPassword = await bcrypt.hash(req.body.password, 8);
+      const queryResult = await userModelManager.create(
```

Comment:
consider changing `queryResult` to `userRecord` because the operation being performed by `userModelManager.create` is not a query operation